### PR TITLE
Copy editing

### PR DESCRIPTION
**Pull request by alexryangregor**
See notes attached to the pull request for "Getting started" for global-level feedback; see notes attached to the pull request for "Register to SEPAL", "Using GEE with SEPAL", and "Recipes" for more context. In short, on this page, I have prioritized grammar, while also ensuring that the content makes sense. I have improved flow, clarity, consistency and quality of the language and readability, when necessary.

### CODE REVIEW

File: docs/source/cookbook/optical_mosaic.rst [hunk 1]
```diff
@@ -15,30 +15,30 @@ This can be achieved on contiguous rasters (left) but also on overlapping images
...
+These overlay areas can be managed in various ways. For example, you can choose to keep only the raster data from the first or last dataset, combine the values of the overlay cells using a weighting algorithm, average the values of the overlay cells, or take the maximum or minimum value. In addition, certain corrections can be made to the image to account for clouds, snow, and other factors. These operations are complex and repetitive. SEPAL offers you an interactive and intuitive way to create mosaics in any area of interest (AOI).
 
-.. warning::
+.. Note::
 
-    You won't be able to retrieve the images if your SEPAL and GEE accounts are not connected. Follow `Connect SEPAL to GEE <../setup/gee.html>`__ to know more.
+    You won't be able to retrieve the images if your SEPAL and Google Earth Engine (GEE) accounts are not connected. Navigate to `Connect SEPAL to GEE <../setup/gee.html>`__ to learn more.
```
Comment (12rambau) [inline docs/source/cookbook/optical_mosaic.rst:22]:
```suggestion
    You won't be able to retrieve the images if your SEPAL and Google Earth Engine (GEE) accounts are not connected. Navigate to :doc:`../setup/gee` to learn more.
```

an inline role that I discovered after writing this page. it makes sure that the title of the link is always the one from the linked page.

File: docs/source/cookbook/optical_mosaic.rst [hunk 2]
```diff
@@ -147,106 +147,106 @@ Date
...
+-   :guilabel:`L8`: `Landsat 8 Tier 1 <https://developers.google.com/earth-engine/datasets/catalog/LANDSAT_LC08_C01_T1>`__. Landsat scenes with the highest available data quality are placed into Tier 1 and considered suitable for time-series processing analysis. Tier 1 includes Level-1 Precision Terrain (L1TP) processed data that have well-characterized radiometry and are inter-calibrated across the different Landsat sensors. The geo-registration of Tier 1 scenes will be consistent and within prescribed tolerances (<=12 m root mean square error [RMSE]). All Tier 1 Landsat data can be considered consistent and inter-calibrated (regardless of the sensor used) across the full collection.
```
Comment (12rambau) [inline docs/source/cookbook/optical_mosaic.rst:178]: this text is actually copy/pasted from the actual satellite documentation, even though it's not perfect, it should not be changed. after links that starts with "developers.google.com, I usually simply copy/pastde the doc, you can discard these lines

File: docs/source/cookbook/optical_mosaic.rst [hunk 3]
```diff
@@ -295,158 +295,158 @@ Cloud masking
...
 
 Analysis
 --------
 
-After selecting the parameters you can start interacting with the scenes and start the analysis.
-On the top right corner, three tabs are available. They will allow you to customize the mosaic scene selection and export the final result.
+After selecting the parameters, you can start interacting with the scenes and begin the analysis.
+In the upper-right corner, three tabs are available. They will allow you to customize the mosaic scene selection and export the final result.
 
 -   :icon:`fas fa-magic`: auto-select scenes
```
Comment (12rambau) [inline docs/source/cookbook/optical_mosaic.rst:337]:
```suggestion
-   :btn:`<fas fa-magic>`: auto-select scenes
```

File: docs/source/cookbook/optical_mosaic.rst [hunk 4]
```diff
@@ -295,158 +295,158 @@ Cloud masking
...
 -   :icon:`fas fa-trash`: clear selected scenes
```
Comment (12rambau) [inline docs/source/cookbook/optical_mosaic.rst:338]:
```suggestion
-   :btn:`<fas fa-trash>`: clear selected scenes
```

File: docs/source/cookbook/optical_mosaic.rst [hunk 5]
```diff
@@ -295,158 +295,158 @@ Cloud masking
...
 -   :icon:`fas fa-cloud-download-alt`: retrieve mosaic
```
Comment (12rambau) [inline docs/source/cookbook/optical_mosaic.rst:339]:
```suggestion
-   :btn:`<fas fa-cloud-download-alt>`: retrieve mosaic
```

**Review by eriklindquist — APPROVED** [review]

**Pull request merged**